Client/Telemetry/TelemetryWrapper.swift

```diff
@@ -671,6 +674,8 @@ extension TelemetryWrapper {
         case shoppingAdsImpression = "shopping-ads-impression"
         case awesomebarShareTap = "awesomebar-share-tap"
         case largeFileWrite = "large-file-write"
+        case fxSuggestionClickInfo = "fx-suggestionClickInfo"
```

Comment:
```suggestion
        case fxSuggestionClickInfo = "fx-suggestion-click-info"
```

Nit, very minor: It doesn't look like any of the other cases are camelCased, so let's match them 🤓